.github/workflows/pull_request.yml

```diff
@@ -47,10 +45,23 @@ jobs:
       run:
         '[[ ${{ matrix.mode }} != "release" ]] || ./ci.sh test'
 
+  # Checks that the AUTHORS files is updated with new contributors.
+  authors:
+    runs-on: [ubuntu-latest]
+    steps:
+    - name: Checkout the source
+      uses: actions/checkout@v2
+      with:
+        # Needs to check out the HEAD of the pull request instead of the merge
+        # and enough history from the pull request to reach the common ancestor.
+        # Normally pull-requests have only 1 commit, so 10 should be enough.
+        ref: ${{ github.event.pull_request.head.sha }}
+        fetch-depth: 10
```

Comment:
@sboukortt  this is not pretty but it will do for now. I think we can do something like always get the merge commit with depth: 2 and compare HEAD with HEAD^; but that's not how we had this set-up and the commit author of the merge is the default email address which is not the one in the commit being merged/rebased.

Reply:
moved this to the ci.sh.